PiConnect/app/src/main/java/com/pi/connect/MainActivity.kt

```diff
@@ -2,16 +2,43 @@ package com.pi.connect
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.recyclerview.widget.LinearLayoutManager
+import kotlinx.android.synthetic.main.activity_main.*
 
 
 class MainActivity : AppCompatActivity() {
-
+    val pythonScripts: ArrayList<String> = ArrayList()
```

Comment:
Remove this and put in the viewModel. But check the warning. Should have been private